### PR TITLE
Confluence integration

### PR DESCRIPTION
**Pull request by khalifapro**
# Confluence Integration✍️

### What
Added Confluence Integration 
​

### CODE REVIEW

**Review by joanagmaia — COMMENTED** [review]
All looks good!
Just two things missing:
You'll also need to add an svg icon to `crowd-icons.svg` file.
You will need to add an 16x16 confluence svg icon (we usually add it with #D1D5DB)
Without this, you might not get the icon displayed in some parts of the app such as default reports

Lint frontend and backend are failing. You can check package.json for the right commands to fix this

**Comment by niravpatel27** [comment]
When can we test this?

**Comment by enkhalifapro** [comment]
@niravpatel27  I wait for an approval to merge the PR. After merge we will need to deploy the branch to staging

**Review by joanreyero — APPROVED** [review]

**Pull request merged**
